cron/views/pages/views.py

```diff
@@ -0,0 +1,34 @@
+from django.views import generic
+from django.views.generic.list import MultipleObjectMixin
+
+from cron.models import Application, Job
+
+
+class ApplicationListView(generic.ListView):
+  model = Application
+  paginate_by = 10
+
+
+class ApplicationDetailView(generic.DetailView, MultipleObjectMixin):
```

Comment:
It won't make a difference here but Mixins should always be placed before the views